vm-setup/roles/common/defaults/main.yml

```diff
@@ -54,6 +54,7 @@ networks:
   - name: baremetal
     bridge: baremetal
     forward_mode: "{% if manage_baremetal == 'y' %}nat{% else %}bridge{% endif %}"
+    enable_natv6: false
```

Comment:
Do you think we need some check when this is true, e.g a task to check the libvirt version, or maybe discover if the feature is available (not sure how discoverable this is, but I guess that would be better).